lsst/ts/laser/csc.py

```diff
@@ -158,7 +191,7 @@ def assert_propagating(self, action):
         -------
 
         """
-        self.model.stop()
+        self.model._laser.clear_fault()
```

Comment:
I think that CSCs have already a standard way of clearing fault states which is to transition from `FAULT` to `STANDBY`. Maybe this command is redundant. Please, consider removing it. 

Regardless of what you decide, this is another coroutine without `await`.... probably can switch to a regular method.

Reply:
I mean you have to actually clear the fault state of the laser otherwise it'll remain that way(this is at the hardware level).

Reply:
Sure. My point is you don't necessarily have to have a command/method to do that. This should probably go in the answer to a `standby` command. So, if there is an error the CSC will transition from `ENABLE` to `FAULT`, and the way to recover it is to send it to `STANDBY` and then re-enable it.